src/components/App.tsx

```diff
@@ -4720,7 +4720,11 @@ class App extends React.Component<AppProps, AppState> {
         pointerDownState.drag.hasOccurred = true;
         // prevent dragging even if we're no longer holding cmd/ctrl otherwise
         // it would have weird results (stuff jumping all over the screen)
-        if (selectedElements.length > 0 && !pointerDownState.withCmdOrCtrl) {
+        if (
```

Comment:
Thanks for the fix @zsviczian! Could you also add a comment here for the check ?

Reply:
done